MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -128,7 +129,9 @@ - (void) handleRenditionChange:(NSNotification *) notif {
     NSNumber *advertisedBitrate = renditionChangeInfo[RenditionChangeNotificationInfoAdvertisedBitrate];
     if (advertisedBitrate) {
         _lastAdvertisedBitrate = [advertisedBitrate doubleValue];
-        [self dispatchRenditionChange];
+        if(![@(_lastDispatchedAdvertisedBitrate) doubleValueIsEqual:@(_lastAdvertisedBitrate)]) {
```

Comment:
I added this since the last review, calling it out. Basically don't dispatch a rendition change event if the bitrate hasn't actually changed.